app/src/main/res/values/strings.xml

```diff
@@ -619,8 +619,12 @@
     <string name="collection_open_tabs">Open tabs</string>
     <!-- Hint for adding name of a collection -->
     <string name="collection_name_hint">Collection name</string>
-    <!-- Text for the menu button to remove a top site -->
-    <string name="remove_top_site">Remove</string>
+	<!-- Text for the menu button to rename a top site -->
+	<string name="rename_top_site">Rename</string>
```

Comment:
When we do string changes, we need to change the string id - that way, the change will get picked up in the localization repo. If we keep the same name, we run the risk of a) string conflicts when we uplift strings to Beta, b) strings not getting re-translated because Pontoon doesn't recognize them as "new strings" (since the id is the same)

When you remove a string, please move it to the `deprecated strings` section of the strings.xml file.

Reply:
Ok, I won't change that.